src/hooks/search/useSearchForm.js

```diff
@@ -0,0 +1,20 @@
+import { useState } from 'react';
```

Comment:
nice, very interesting implementation, also I like that you extracted this logic from the component